### ISSUE TITLE
Issues with .ogg song files

### ISSUE THREAD

**Issue by Vaarelsauce** [EnanoFurtivo/Osu-Mp3-Extractor#3]
It's been working perfectly fine so far for me, but every once in a while I get a song that uses .ogg instead of .mp3 in the source folder. When it's extracted, the file that comes out is listed as .mp3, but I don't think it actually is (or at least is corrupted). 

I can play it perfectly fine, BUT I:
- can't edit the metadata properties, 
- mp3-troubleshooter programs telling me that it is indeed corrupted
- can't push those files into iTunes music, for portability

As a temporary solution, I've painstakingly singled out those specific song's source folder, and manually exported them with Modricagon's 2018 music exporter (which isn't as good as yours, but can probably get the job done).
EDIT: Their program doesn't even work with .ogg files either. Facepalm.

**Comment by EnanoFurtivo** [comment]
**TLDR; You might want to try modifying the extension from .mp3 back to .ogg**

This error is due to my inexperience in the past. When the file is copied the extension .mp3 is forced in the file name, this is a programming mistake and the original extension should be kept.

When checking the code uploaded here, the version does not match the executable that is uploaded and works (the one you are probably using). This is because I started to refactor this code about 2 years ago. If renaming the file back to .ogg does not work then you might have to wait until I finish refactoring this app, which I don't know when it will happen. I'm very sorry for this.
